packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -229,7 +236,24 @@ Function StatusAll() {
     Write-Output "  `"status`": `"${cwa_status}`","
     Write-Output "  `"starttime`": `"${cwa_starttime}`","
     Write-Output "  `"configstatus`": `"${cwa_config_status}`","
-    Write-Output "  `"version`": `"${version}`""
+    
+    if ($IncludeWorkloads) {
+        $workloads_json = "[]"
+        $workload_discovery_path = $WORKLOAD_DISCOVERY
+        if (Test-Path -LiteralPath $workload_discovery_path) {
+            try {
+                $workloads_json = cmd /c "`"$workload_discovery_path`" 2>null"
+                if (-not $workloads_json) { $workloads_json = "[]" }
+            } catch {
+                $workloads_json = "[]"
+            }
+        }
+        Write-Output "  `"version`": `"${version}`","

```

Comment:
why are we moving `Write-Output " `"version`": `"${version}`""` should this if statement just print workloads or not, I don't this statement should effect the version

Reply:
The `status` action and `status-with-workloads` action is using a combined function, so that's why it was moved.